docs/source/advanced_features/cloud_storage.rst

```diff
@@ -57,3 +57,86 @@ Create a dataset with authorized cloud storage:
 .. important::
 
     The directory ``path/to/dataset`` should be empty when create an authorized storage Fusion Dataset.
+
+Inport data into tensorby
+=========================
+
+If your use cloud platform to storage data, you will first need to import your data into TensorBay.
+
+There are two options to import your data from raw cloud platform path to authorized storage dataset.
+
+1. Import all files with a directory path into datset -- out-of-the-box
+2. Use AuthData to organized a dataset -- customization
+
+Suppose the cloud storage platform structure like ::
+
+   datas/
+   ├── images/
+   │   ├── 00001.png
+   │   ├── 00002.png
+   │   └── ...
+   ├── labels/
+   │   ├── 00001.json
+   │   ├── 00002.json
+   │   └── ...
+   └── ...
+
+
+Import all files from one directory into a target segment.
+**********************************************************
+
+Import all files in `datas/images` directory into `train` segment.
+
+.. code:: python
+
+   dataset_client.import_all_files("datas/images", "train")
+
+
+Import files and upload label to several segments
+*************************************************
+
+Use cloud client to interact with cloud platform.
+
+Get a cloud client.
+
+.. code:: python
+
+    cloud_client = dataset_client.get_cloud_client()
+
+List all files in `datas/train`
+
+.. code:: python
+
+    files = cloud_client.list_files("datas/images")
+    files[0] # "datas/images/00001.png"
+
+open one file in clout platform
+
+.. code:: python
+
+    datas = cloud_client.open("datas/labels/0001.json").read()
+
+Use the file path in cloud platform as the AuthData path.
+
+.. code:: python
+
+   AuthData("train/data/0001.png")
+
+load label file from cloud platform to tensorby label format.
+
+.. literalinclude:: ../../../docs/code/cloud_storage.py
+   :language: python
+   :start-after: """Load label file from cloud platform into the AuthData"""
+   :end-before: """"""
+
+Import dataset from cloud platform to authorized storage dataset.
+
+.. literalinclude:: ../../../docs/code/cloud_storage.py
+   :language: python
+   :start-after: """Import dataset from cloud platform to the authorized storage dataset"""
+   :end-before: """"""
+
+.. important::
+
+    The file will copy from raw directory to the authorized cloud storage dataset path,
+    set `delete_source=True` to delete raw files when the import is finished.
```

Comment:
1. Please use double apostrophes like ``delete_source=True`` to describe inline code.
There are several cases in this PR, please correct them all.
2. Please describe explicitly the place to set delete_source=True. 
Add a line of code, for example.